test/monitoring/component-alerts_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build aws packet
+// +build poste2e
+
+package monitoring
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+)
+
+type AlertTestCase struct {
+	Name      string
+	Platforms map[string]interface{}
+	Alerts    []string
+}
+
+// nolint:funlen
```

Comment:
do we really need this?

Reply:
Yes we need this otherwise the linter complains.